### PR TITLE
feat: add plugin-site-issues chart

### PR DESCRIPTION
**Pull request by halkeye**
https://github.com/jenkins-infra/public-charts/pull/31
https://github.com/jenkins-infra/charts-secrets/pull/20
https://github.com/jenkins-infra/charts/pull/1790

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -11,6 +11,9 @@ reports.yaml @jenkins-infra/kubernetes
 plugin-site/ @jenkins-infra/kubernetes @jenkins-infra/plugin-site
 plugin-site.yaml @jenkins-infra/kubernetes @jenkins-infra/plugin-site
 
+plugin-site-issues/ @jenkins-infra/kubernetes @jenkins-infra/plugin-site
+plugin-site-ssues.yaml @jenkins-infra/kubernetes @jenkins-infra/plugin-site
```
Comment (lemeurherve) [inline CODEOWNERS:15]:
```suggestion
plugin-site-issues.yaml @jenkins-infra/kubernetes @jenkins-infra/plugin-site
```

File: .sops.yaml [hunk 2]
```diff
@@ -55,3 +55,17 @@ creation_rules:
...
+            key: "sops"
+            version: "252fc7e646ac40f184ab4a6016484725"
+        pgp:
+          - "88FABF5F307FB5870B6AD2E8A266F2D3F9D60C45" # Halkeye
+          - "6699E555C6730CAED9083B7AD40F4AD2F55AF15F" # timja
+          - "56D8342434B84E2D1CCF53D96E9A025D52210D3D" # Olblak
+          - "29B84443F41DE582F71599AFB47082DEE225AE06" # dduportal
+          - "81B0C54A1BA2C92EAD985025A2B9560AF24FF1AD" # hlemeur
+          - "E104B7543ECCCC68EA1EB35DD9B2DBFB59AD8344" # markewaite
+          - "56D8342434B84E2D1CCF53D96E9A025D52210D3D" # Olblak
```
Comment (lemeurherve) [inline .sops.yaml:71]:
```suggestion
    - path_regex: secrets/config/plugin-site-issues/secrets.yaml
      key_groups:
        - azure_keyvault:
            - vaultUrl: "https://prodjenkinsinfra.vault.azure.net"
              key: "sops"
              version: "252fc7e646ac40f184ab4a6016484725"
          pgp:
            - "88FABF5F307FB5870B6AD2E8A266F2D3F9D60C45" # Halkeye
            - "6699E555C6730CAED9083B7AD40F4AD2F55AF15F" # timja
            - "56D8342434B84E2D1CCF53D96E9A025D52210D3D" # Olblak
            - "29B84443F41DE582F71599AFB47082DEE225AE06" # dduportal
            - "81B0C54A1BA2C92EAD985025A2B9560AF24FF1AD" # hlemeur
            - "E104B7543ECCCC68EA1EB35DD9B2DBFB59AD8344" # markewaite
            - "56D8342434B84E2D1CCF53D96E9A025D52210D3D" # Olblak
```

**Review by lemeurherve — CHANGES_REQUESTED** [review]

File: clusters/publick8s.yaml [hunk 3]
```diff
@@ -127,6 +127,14 @@ releases:
       - "../config/plugin-site.yaml"
     secrets:
       - "../secrets/config/plugin-site/secrets.yaml"
+  - name: plugin-site-issues
+    namespace: plugin-site
+    chart: jenkins-infra/plugin-site-issues
+    version: 0.1.0
```
Comment (lemeurherve) [inline clusters/publick8s.yaml:133]:
```suggestion
    version: 0.1.1
```

**Review by lemeurherve — CHANGES_REQUESTED** [review]

File: config/plugin-site-issues.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  className: public-nginx
+  annotations:
+    "cert-manager.io/cluster-issuer": "letsencrypt-prod"
+    "nginx.ingress.kubernetes.io/ssl-redirect": "true"
+    "nginx.ingress.kubernetes.io/enable-cors": "true"
+    "nginx.ingress.kubernetes.io/cors-allow-methods": "GET, OPTIONS"
+    "nginx.ingress.kubernetes.io/cors-allow-origin": "*"
+  hosts:
+    - host: plugin-site-issues.origin.jenkins.io
+    - host: plugin-site-issues.jenkins.io
```
Comment (lemeurherve) [inline config/plugin-site-issues.yaml:12]:
```suggestion
    - host: plugin-site-issues.origin.jenkins.io
      paths:
        - path: /
          pathType: Prefix
    - host: plugin-site-issues.jenkins.io
      paths:
        - path: /
          pathType: Prefix
```

Comment (halkeye) [reply config/plugin-site-issues.yaml:12]: I suspect strongly we won't even need origin, but sounds good

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**